float32 press
float32 in_temp
float32 in_humi
float32 out_temp
float32 out_humi
float32 wind_sp
float32 wind_dir
float32 rain_rate
string error_check

